src/index.ts

```diff
@@ -9,6 +9,16 @@ export {
   CommandMatcher,
 } from './command/command';
 
+import { Alice } from './alice';
```

Comment:
I believe we need separate commit or PR for this.

Reply:
excluded in a separate PR